gatsby/src/components/Footer/Footer.test.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { render } from "@testing-library/react";
+
+import { Footer } from "./Footer";
+
+describe("Footer", () => {
+	it("should render global nav", async () => {
+		const { findByText } = render(<Footer />);
+
+		expect(await findByText("NICE Pathways")).toBeInTheDocument();
```

Comment:
i like this - no more nextTick() hopefully...

Reply:
Yeah it's a lot more elegant